test_data/yast/encryption/crypt_no_lvm_enc_luks2.yaml

```diff
@@ -0,0 +1,23 @@
+disks:
```

Comment:
Please remove the `disks` and `lvm` section, they are wrong here.

Reply:
Removed and run a VR https://openqa.opensuse.org/tests/4223858#